src/horizontal.jsx

```diff
@@ -50,19 +51,13 @@ export default function(defaultGutter, gutterMultiplier, defaultGutterUnit) {
         height: this.props.height
       };
 
-      // Flex with vendor prefixes:
       // display:flex
-      styles.display = prefixDisplayFlex();
+      // done through class name so we get vendor prefixes
+
       // flex-direction
-      styles.WebkitBoxOrient = 'horizontal';
-      styles.WebkitBoxDirection = 'normal';
-      styles.WebkitFlexDirection = 'row';
-      styles.msFlexDirection = 'row';
-      styles.flexDirection = 'row';
+      // done through class name so we get vendor prefixes
       // flex-wrap
-      styles.WebkitFlexWrap = 'nowrap';
-      styles.msFlexWrap = 'nowrap';
-      styles.flexWrap = 'nowrap';
+      // done through class name so we get vendor prefixes
       // justify-content
       styles.WebkitBoxPack = cssValueToOldFlexSyntax(justifyItems);
```

Comment:
if you want to go another step further you can change this to use a set of classnames and take advantage of browser prefixing in CSS